src/components/UI/TeamList.vue

```diff
@@ -0,0 +1,64 @@
+<script setup>
+// Components
+import PageLayout from '@/components/UI/PageLayout.vue'
+import Card from '@/components/UI/Card.vue'
+import TextBlock from '@/components/UI/TextBlock.vue'
+import Button from '@/components/UI/Button.vue'
+// const props = defineProps({})
+// const emit = defineEmits()
+
+const title = 'Team List'
+const team = [
+    {
+        name: 'Dill',
+        title: 'Dumbass'
+    },
+    {
+        name: 'Pickle',
+        title: 'Trash Annihalator'
+    },
+    {
+        name: 'Brandy',
+        title: 'Goblin'
+    },
+    
+]
+
+</script>
+<template>
+    <PageLayout>
+        <template #main-content>
+            <section class="mt-8 w-10/12 h-dvh">
+                <Card>
+                    <section class="flex-col w-1/2 text-left">
+                        <TextBlock 
+                            :text-content="title" 
+                            contentClass="text-2xl font-bold"
```

Comment:
its a vue convention to have these prop names hyphenated, just a little tweak there.